include/deal.II/numerics/solution_transfer.h

```diff
@@ -113,6 +113,35 @@ DEAL_II_NAMESPACE_OPEN
  * soltrans.interpolate(solution, interpolated_solution);
  * @endcode
  *
+ * <li> If the grid is distributed, or partitioned across several MPI processes,
```

Comment:
Should this discussion not be part of the `parallel::distributed::SolutionTransfer` class?

Reply:
As you can see from the tests its relevant for the standard `SolutionTransfer` class when using `Triangulation` + `Metis`/`Zoltan` as partitioners. So I think that it should stay here, but maybe with a clarification of that point. Would you like me to copy it over to the `p::d::ST` documentation as well?

Reply:
Yes, maybe. Thanks!

Is there any mention of `p::d::ST` anywhere here at all?

Reply:
@bangerth Sorry its taken me so long to get to this. I've now added a blurb in the introduction to `p::d::ST`.